src/middlewared/middlewared/main.py

```diff
@@ -823,18 +823,21 @@ def add_schema(self, schema):
     def get_schema(self, name):
         return self.__schemas.get(name)
 
-    async def threaded(self, method, *args, **kwargs):
+    async def _threaded(self, pool, method, *args, **kwargs):
```

Comment:
How about renaming these to `self.run_in_thread` and `self.run_in_thread_pool`?

Reply:
`threaded` is already in use by many things, I did not want to change that.